mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
It is only used in one place that is not a debug message or internal to the PageSettings dialog box (pagesettings.cpp).  And that one place is here:
https://github.com/musescore/MuseScore/pull/2573/files/831df406b6bd81c4e3a2f79c4551410470f343c0#diff-53fdc139c4fd1972f05706a81c9ee7c0R100